hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -94,7 +116,171 @@ def get_data_module(self) -> PandaTilesDataModule:
         )
 
     def get_slides_dataset(self) -> Optional[PandaDataset]:
-        return PandaDataset(root=self.local_datasets[1])                             # type: ignore
+        return PandaDataset(root=self.local_datasets[1])  # type: ignore
+
+
+class DeepSMILETilesPanda(DeepSMILETilesPandaBasic):
```

Comment:
1. I would definitely recommend putting the benchmark config, whatever is called, in a different module. This would make it easier to find for the user. 
2. Technically - the benchmark config to reproduce Myronenko et al. 2021 should be the one that starts from slides (therefore inherits from DeepSMILESlidesPanda)